### DISCUSSION TITLE
ASUSTOR AS5202T Install question

### DISCUSSION THREAD

**Discussion by drew88syn** [Taxel/PlexTraktSync#757] [Support] [answered]
Hi!
im running my plex server on an ASUSTOR AS5202T NAS
Whats the easiest way to install plextraktsync on this NAS?

**Comment by stevietv** [comment]
what OS is running on the NAS, how is plex running.
Without this basic information its hard to advise :)

**Reply by drew88syn** [reply]
Its running on an Intel x64 based ADM OS
PLEX is installed from the asustor store.
I can access docker trough portainer

**Comment by stevietv** [comment, marked as answer]
then you likely have to look into how you can install python3 on Asustor
and run a python script
